spec/index.html

```diff
@@ -897,7 +918,7 @@ <h2>Anchors and Aliases</h2>
       (and eventually the in-memory representation
       of the data structure, e.g., a Python dictionary or a Java hashmap)
       will still contain references between nodes,
-      the JSON-LD serialization will not — since, by that moment,
+      the JSON-LD serialization will not — since, by the moment when it is formed,
       all the anchors are resolved, as shown below.
```

Comment:
```suggestion
      the JSON-LD serialization will not — since, by the time it is formed,
      all the anchors have been resolved, as shown below.
```